examples/event-source-mapping/main.tf

```diff
@@ -216,21 +226,26 @@ resource "aws_kinesis_stream" "this" {
 }
 
 # Amazon MQ
-data "aws_vpc" "default" {
-  default = true
-}
+module "vpc" {
```

Comment:
I don't have any default VPCs, so added a bare bones one to make it work for the RabbitMQ requirements